utils/mediaupload.js

```diff
@@ -118,6 +121,7 @@ export function mediaUpload( {
 					caption: get( savedMedia, [ 'caption', 'raw' ], '' ),
 					id: savedMedia.id,
 					link: savedMedia.link,
+					mime_type: savedMedia.mime_type,
```

Comment:
can certainly make the change, but I'm not sure how this will impact anything dependent. Does anything outside this consume `mime_type` as it is defined here?

Reply:
Oh, we can rename. this was added in for playlists. fixed.